.circleci/config.yml

```diff
@@ -87,6 +88,32 @@ jobs:
             source env/bin/activate
             python manage.py test
 
+  staging_deploy:
+    docker:
+      - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
+        environment:
+          GITHUB_TEAM_NAME_SLUG: laa-get-access
+          APPLICATION_DEPLOY_NAME: laa-cla-public
+    steps:
+      - checkout
+      - run:
+          name: Kubectl deployment
+          command: |
+            setup-kube-auth
+            kubectl config use-context staging
+      - deploy:
+          name: Deploy laa-cla-public docker image
+          command: |
+            safe_git_branch=${CIRCLE_BRANCH//\//-}
+            short_sha="$(git rev-parse --short=7 $CIRCLE_SHA1)"
+
+            kubectl set image --filename=kubernetes_deploy/staging/deployment.yml --local --output=yaml \
+              app=${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${APPLICATION_DEPLOY_NAME}:$safe_git_branch.$short_sha | \
+              kubectl apply \
+                --filename=- \
```

Comment:
`-` might be a bit cryptic to some developers. Is it worthwhile using `/dev/stdin`?